utilix/batchq.py

```diff
@@ -116,13 +121,13 @@ class JobSubmission(BaseModel):
         False, description="Exclude the loosely coupled nodes"
     )
     log: str = Field("job.log", description="Where to store the log file of the job")
+    partition: Literal[
```

Comment:
Why this is needed? https://github.com/XENONnT/utilix/pull/110/commits/82a36ce65ceacd7c4c831f154dae83a257fcbbb5

Reply:
That's because the sequence that pydantic validates the inputs is based on the sequence of the arguments but not the sequence of the validators to be defined. As we need to override the bind for dali, it's better to validate the bind after partition.